src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -176,18 +176,18 @@ def generate_FAD_2_default_config(self, filename):
     }
     # Generate FAD Records for each record type like cbsd,zone and esc_sensor
     cbsd_fad_records = generateCbsdRecords(cbsd_records, grant_record_list)
-    ppa_fad_record = generatePpaRecords(ppa_records, cbsd_reference_ids)
-    esc_fad_record = [esc_sensor]
+    ppa_fad_records = generatePpaRecords(ppa_records, cbsd_reference_ids)
```

Comment:
Please use CBSD reference IDs which have a valid format so that the SAS UUT doesn't fail during data validation.

Reply:
Updated   the CBSD Reference ID format.